umpost/conversion_driver_esm1p5.py

```diff
@@ -295,25 +296,36 @@ def safe_removal(succeeded, failed):
     return succeeded_inputs - failed_inputs
 
 
-if __name__ == "__main__":
+def parse_args(args):
+    """
+    Parse arguments given as list (args)
+    """
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "current_output_dir", help="ESM1.5 output directory to be converted", type=str
+        "current_output_dir", help="ESM1.5 output directory to be converted",
+        type=str
     )
     parser.add_argument("--quiet", "-q", action="store_true",
                         help=(
-                            "Report only final exception type and message for allowed"
-                            "exceptions raised during conversion when flag is included."
-                            "Otherwise report full stack trace."
+                            "Report only final exception type and message for "
+                            "allowed exceptions raised during conversion when "
+                            "flag is included. Otherwise report full "
+                            "stack trace."
                         )
                         )
     parser.add_argument("--delete-ff", "-d", action="store_true",
                         help="Delete fields files upon successful conversion."
                         )
-    args = parser.parse_args()
+
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args(sys.argv[1:])
```

Comment:
The `sys.argv[1:]` arg can be removed with the change to `parse_args()`.

Reply:
Have removed this one too